python/ingestion/cdc_wonder_utils.py

```diff
@@ -0,0 +1,263 @@
+
+            count_cols_to_sum = [COUNT_COL, POP_COL]
+
+            topic_df = combine_race_ethnicity(
+                topic_df,
+                count_cols_to_sum=count_cols_to_sum,
+                race_alone_to_het_code_map=BREAKDOWN_TO_STANDARD_BY_COL[std_col.RACE_CATEGORY_ID_COL],
+                ethnicity_value="Hispanic",
+            )
+
+        topic_df = rename_cols(
+            topic_df,
+            cast(GEO_TYPE, geo_level),
+            cast(CANCER_TYPE_OR_ALL, breakdown),
+            condition,
+            state_code_col,
+        )
+
+        topic_dfs.append(topic_df)
+
+    df_merged = merge_dfs_list(topic_dfs, merge_cols)
+
+    # drop rows that dont include FIPS and DEMO values
+    df_merged = df_merged[df_merged[std_col.STATE_FIPS_COL].notna()]
+    df_merged = dataset_utils.ensure_leading_zeros(df_merged, std_col.STATE_FIPS_COL, 2)
+
+    return df_merged
+
+
+def rename_cols(
+    df: pd.DataFrame,
+    geo_level: GEO_TYPE,
+    breakdown: CANCER_TYPE_OR_ALL,
+    condition: str,
+    state_code_col: str,
+) -> pd.DataFrame:
+    """Renames columns based on the demo/geo breakdown"""
+
+    # Create consistent column names for cancer data
+    rename_cols_map: Dict[str, str] = {
+        COUNT_COL: f'{condition.lower()}_count_{std_col.RAW_SUFFIX}',
+        POP_COL: f'{condition.lower()}_{std_col.RAW_POP_SUFFIX}',
+        CRUDE_RATE_COL: f'{condition.lower()}_{std_col.PER_100K_SUFFIX}',
+    }
+
+    if geo_level in [STATE_LEVEL, NATIONAL_LEVEL]:
+        rename_cols_map[state_code_col] = std_col.STATE_FIPS_COL
+
+    if breakdown == std_col.RACE_OR_HISPANIC_COL:
+        rename_cols_map[RACE_NAME] = std_col.RACE_CATEGORY_ID_COL
+
+    if breakdown == std_col.AGE_COL:
+        rename_cols_map[AGE_GROUP] = std_col.AGE_COL
+
+    if breakdown == std_col.SEX_COL:
+        rename_cols_map[SEX_COL] = std_col.SEX_COL
+
+    df = df.rename(columns=rename_cols_map)
+    return df
+
+
+def get_age_adjusted_ratios(df: pd.DataFrame, conditions: List[str]) -> pd.DataFrame:
+    """Adds columns for crude rate ratios (comparing each race's
+    rate to the rate for White NH) for each type of cancer."""
+
+    _tmp_white_rates_col = 'WHITE_NH_RATE'
+
+    for condition in conditions:
+        cancer_type = condition.lower()
+        source_rate_col = f'{cancer_type}_{std_col.PER_100K_SUFFIX}'
+        het_ratio_col = f'{cancer_type}_{std_col.RATIO_AGE_ADJUSTED_SUFFIX}'
+
+        white_nh_rates = df[df[std_col.RACE_CATEGORY_ID_COL] == std_col.Race.WHITE_NH.value].set_index(
+            std_col.STATE_FIPS_COL
+        )[source_rate_col]
+
+        df[_tmp_white_rates_col] = df[std_col.STATE_FIPS_COL].map(white_nh_rates)
+        df[het_ratio_col] = df[source_rate_col] / df[_tmp_white_rates_col]
+        df[het_ratio_col] = df[het_ratio_col].round(2)
+
+        df = df.drop(columns=[_tmp_white_rates_col])
+        df.loc[df[std_col.RACE_CATEGORY_ID_COL] == std_col.Race.ALL.value, het_ratio_col] = np.nan
+
+    return df

```

Comment:
are you using this function? if so, is it really age-adjusting? or are you reading in the source data pre-age adjusted?